README.md

```diff
@@ -149,7 +149,40 @@ Repeat the steps above for all running NuoDB engine containers you want to monit
 
 # Setup in Kubernetes
 
-NuoDB Helm Chart support for the NuoDB Collector coming soon.
+## Software Release requirements
+
+| Software   | Release Requirements                           | 
+|------------|------------------------------------------------|
+| NuoDB Helm | NuoDB Helm Charts [3.0.0](https://github.com/nuodb/nuodb-helm-charts) or newer |
+
+
+## Deploying NuoDB Collector using NuoDB Helm Charts
+
+NuoDB Collector will be deployed in sidecar containers using NuoDB Helm Charts.
+
+See instructions for [NuoDB Helm charts](https://github.com/nuodb/nuodb-helm-charts/blob/master/README.md#nuodb-helm-chart-installation) installation.
+NuoDB Collector can be enabled separately for [Admin](https://github.com/nuodb/nuodb-helm-charts/stable/admin) and [Database](https://github.com/nuodb/nuodb-helm-charts/stable/database) charts. To enable it set the `insights.enabled` variable to `true`. For example:
+
+```bash
+helm install admin nuodb/admin --set insights.enabled=true --namespace nuodb
```

Comment:
we use a lot of terms interchangeably. But you are not enabling "Insights - the monitoring tool". You are enabling nuocd (nuodb-collector) - the stats collector. Maybe we need to rename that variable.

Reply:
At some point, I also considered changing the value to something like `nuocd` but didn't do it because we already had `insights` value in the past. Could it be possible to place a comment on https://github.com/nuodb/nuodb-helm-charts/pull/161 as there is a bigger audience there? If we agree on a change, then I will update the docs as well.

Reply:
Updated to `nuocollector` as agreed.